cassandra-analytics-core/src/main/java/org/apache/cassandra/spark/bulkwriter/RecordWriter.java

```diff
@@ -125,7 +137,6 @@ public List<StreamResult> write(Iterator<Tuple2<DecoratedKey, Object[]>> sourceI
         // for all replicas in this partition
         validateAcceptableTimeSkewOrThrow(new ArrayList<>(instancesFromMapping(initialTokenRangeInstances)));
 
-        LOGGER.info("[{}]: Processing bulk writer partition", taskContext.partitionId());
```

Comment:
Why is the line removed? I think we are fine with more 1 message per spark partition.

Reply:
Not removing it. The diff likely has it as a new line.

Reply:
Got it. It was moved to the beginning of the method.